content/about/at-support-tables/at-support-tables.html

```diff
@@ -0,0 +1,122 @@
+        Failure to provide a “Should-Have” behavior could impede users.
+        Learn more about ARIA-AT’s
+        <a href="https://github.com/w3c/aria-at/wiki/Glossary#assertion-priority">definitions of Must and Should on the project wiki</a>.
+        </p>
+
+      <section>
+        <h3>Examples of Must-Have Behaviors</h3>
+        <ul>
+          <li>Convey the name of a radio button.</li>
+          <li>Convey the state of a checked radio button.</li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>Examples of Should-Have Behaviors</h3>
+        <ul>
+          <li>Convey the position of a radio button in a radio group, e.g., the button is 1 of 3.</li>
+          <li>Convey the number of radio buttons in a radio group.</li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>Important Constraints</h3>
+        <ul>
+          <li>Unless otherwise noted, all testing is done using the assistive technology vendor's default configuration of an assistive technology.</li>
+          <li>
+            ARIA-AT interoperability  tests do not prescribe exactly how to satisfy a need.
+            For example, they do not specify exactly what a screen reader should speak.
+            Two different screen readers may convey the same information in different ways.
+          </li>
+        </ul>
+      </section>
+    </section>
+
+    <section id="recommendations">
+      <h2>Recommendations</h2>
+
+      <section>
+        <h3>Don’t Code to the Bugs</h3>
+        <p>
+          ARIA-AT is working with assistive technology vendors to increase their support levels.
+          This means that assistive technologies that align with ARIA-AT interoperability tests will change over time.
+          Exercise caution when implementing a pattern where support levels are less than 100%.
+          Avoid modifying code to accommodate an assistive technology failure unless you are confident that the modification will:
+        </p>
+        <ul>
+          <li>Not negatively affect the experience when using assistive technologies that provide 100% support.</li>
+          <li>Not negatively affect the experience when the assistive technology vendor resolves the failure being accommodated.</li>
+        </ul>
+        <p>
+          When possible, test implementations of APG patterns with an assistive technology that provides 100% support for both must-have and should-have behaviors.
+        </p>
+      </section>
+
+      <section>
+        <h3>Design to Mitigate Critical Support Failures</h3>
+        <p>
+          Where feasible, avoid designing experiences that rely on features of APG patterns that have less than 100% support for must-have behaviors.
+          If the must-have support level is less than 100% for the example implementation of a pattern, that does not mean all ways of implementing that pattern will present assistive technology users with critical problems.
```

Comment:
@mcking65 Can we change "all ways of implementing that pattern" with "multiple ways of implementing that pattern" which was used in #1 suggestion below? 

"If there are multiple implementation examples of..."

Reply:
that would change the meaning. I revised the wording a bit to hopefully resolve the lack of clarity ...

>If the must-have support level is less than 100% for one example implementation of a pattern, that does not mean every other way of implementing that pattern will present assistive technology users with the same problems.